CHANGELOG.md

```diff
@@ -2,6 +2,94 @@
 
 ## Release 2025-01-06
 
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as `v20250130`.
+
+### Announcements
+* General support for AKS Kubernetes version 1.28 was deprecated on Jan 30, 2025. Upgrade your clusters to version 1.29 or later. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information.
+* Security patch releases in release tracker, starting with 20250115T000000Z will contain release notes for the release.
+
+### Release Notes
+* Features:
+  * AKS Kubernetes patch versions 1.29.11, 1.30.7 and 1.31.3 are now available.
+  * AKS LTS version 1.28 available in all regions.
+  * The upcoming apiVersion 2025-01-01 will enable additional outbound configurations to allow for servicing of network isolated clusters.
+  * CNI validation for node autoprovisioner now allows all CNI configurations except for Calico and kubenet.
+  * AKS Automatic SKU now supports BYO Virtual Networks.
+
+* Behavior change:
+  * Proper casing will be enforced on PUT of `Microsoft.ContainerService/managedClusters/agentPools` for the `AgentPoolMode` property. See this [issue](https://github.com/Azure/AKS/issues/4468) for more detail.
+  * Change the toggle for cilium nodesubnet support to `true` by default. The feature is still behind a feature flag  `EnableCiliumNodeSubnet`, but with this change, customers can create cilium nodesubnet clusters.
+  * Changes to removes Prometheus port and scrape annotations from Retina Linux and Windows daemonset for basic and advanced. This avoids duplication for customers utilizing Retina.
+  * New clusters will no longer be able to enable the HTTP application routing add-on. See the notice on this migration guide: https://learn.microsoft.com/en-us/azure/aks/app-routing-migration
```

Comment:
Reword as:
The HTTP Application Routing add-on (preview) is now retired. You can no longer create clusters that enable the add-on. [Migrate](https://learn.microsoft.com/Azure/AKS/app-routing-migration) to the generally available Application Routing add-on.

Reply:
updated and moved to announcements